zhaquirks/xiaomi/aqara/light_acn.py

```diff
@@ -200,3 +224,87 @@ class LumiLightAcn014(XiaomiCustomDevice):
             },
         }
     }
+
+
+class LumiLightAcn032(XiaomiCustomDevice):
+    """Aqara Light T1M (ceiling light with RGB ring).
+
+    This quirk adds support for power on behavior by adding the power_outage_memory attribute, the rest works out
+    of the box.
+
+    It MIGHT also work for the Model T1 (acn031) but it has not been tested.
+    """
+
+    signature = {
+        MODELS_INFO: [("Aqara", "lumi.light.acn032"), ],
+        ENDPOINTS: {
+            1: {
+                PROFILE_ID: zha.PROFILE_ID,
+                DEVICE_TYPE: zha.DeviceType.COLOR_DIMMABLE_LIGHT,
+                INPUT_CLUSTERS: [
+                    BasicCluster.cluster_id,  # 0x0000
```

Comment:
Can you remove these comments at the end everywhere? We don't really use them in our repo and it should be kept consistent.